site/docs/v1/tech/installation-guide/cluster.adoc

```diff
@@ -125,6 +122,25 @@ Here's a video covering how to add and remove nodes from a FusionAuth cluster:
 
 video::y6bPjqP4Dzk[youtube,width=560,height=315]
 
+==== "Bye node" Messages
+
+There are two different levels of cluster membership. The first is managed by the load balancer and concerns what traffic is sent to which node. FusionAuth operates a second level of cluster membership for the limited state shared between nodes.
+
+include::docs/v1/tech/installation-guide/_shared-state.adoc[]
+
+Each node regularly updates the shared database by updating a row with URL and timestamp information. If a node does not check in, after a certain period it will be removed from the cluster, as far as FusionAuth is concerned.
+
+If that happens you might see a message like this:
+
+[source]
+----
+io.fusionauth.api.service.system.NodeService - Node [abce451c-6c5f-4615-b4eb-c1ae5ccf460c] with address [http://10.0.0.2:9011] removed because it has not checked in for the last [83] seconds. Bye node.
+----
+
+While a node is removed from FusionAuth's node list, it will no longer participate in the FusionAuth cluster actions as mentioned above.
+
+This automated removal does not affect load balancer traffic. The load balancer, typically by using a health check, must stop sending a node authentication traffic if it is unhealthy.
```

Comment:
Yes, this is good we want to assure them that this does not mean the node is no longer participating. 

If the node is actually un-healthy then it may actually be dead and not participating, or it may just be that the node is super busy and we can't get a connection to the db to ping ourselves. If this is the case, once the load lets up, it will check in, and be added back to the node list. 

This is sort of all internal stuff, so in general the "end user" can assume this is all just informational messages in the logs.